gaj23/tic-tac-talon
# Gaj/refactor/linter

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. Include screenshots as needed.

This adds GitLab CI to my project. Additionally, eslint was added and adjustments were made based on that feedback. 

Issues Closed #17 #13 #15

### Problems Encountered & Resulting Solutions  
> State any problems encountered & changes made.

NA

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Please review the yml file to see CI rules. 

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please check the CI pipelines & load the gh-pages to see if there's any warnings. 

### Steps To Take
> Any questions and possible steps moving forward.

Move on to Responsive Design
